src/components/EditWebhooks.js

```diff
@@ -189,48 +229,57 @@ class EditWebhooks extends React.Component {
       intl,
       data: { loading },
     } = this.props;
-
-    let submitBtnMessageId = 'save';
-    if (['loading', 'saved'].includes(status)) {
-      submitBtnMessageId = status;
-    }
-    const submitBtnLabel = this.messages[submitBtnMessageId] && intl.formatMessage(this.messages[submitBtnMessageId]);
+    const webhooksCount = webhooks.length;
 
     return loading ? (
       <Loading />
     ) : (
       <div className="EditWebhooks">
-        <style jsx>
-          {`
-            .error {
-              color: red;
-            }
-          `}
-        </style>
-
-        <div className="webhooks">
-          <h2>{this.props.title}</h2>
-          {webhooks.map(this.renderWebhook)}
-        </div>
-
-        <div className="editWebhooksActions">
-          <Button bsStyle="primary" onClick={() => this.addWebhook()}>
+        <h2>{this.props.title}</h2>
+        <StyledHr />
+
+        <div className="webhooks">{webhooks.map(this.renderWebhook)}</div>
+
+        {webhooksCount > 0 && <StyledHr />}
+
+        <Box width={[0.9, 1.5 / 2]} mx={['auto', 0, 0]} my={3} className="editWebhooksActions">
```

Comment:
Why `1.5 / 2` and not `0.75` (to match the format of `0.9` next to it)?

Reply:
I was counting base on the view scale of 1x 2x 3x. 
Yah, I would change it. It's more readable at 0.75.